core/src/toga/widgets/box.py

```diff
@@ -29,17 +29,17 @@ def __init__(
 
     @property
     def enabled(self):
-        """Is the widget currently enabled? i.e., can the user interact with the
-        widget?
+        """Is the widget currently enabled? i.e., can the user interact with the widget?
```

Comment:
I'm not clear why *this* didn't get split into 2 lines where the implementation on base did.

Reply:
hmm....my guess is because it has an empty line and more text following. So, I guess the "one sentence on first line" rule isn't being enforced here....